src/Maestro/maestro-angular/src/app/page/build/build.component.html

```diff
@@ -101,18 +101,22 @@ <h5 class="card-title float-left">
         <ng-container *stateful="let graph from graph$; loadingTemplate: loadingData">
           <ul class="nav nav-tabs">
             <li class="nav-item">
-              <a [routerLink]="['..', 'graph']" routerLinkActive="active" style="cursor: pointer;" class="nav-link">Graph</a>
+              <a [routerLink]="['..', 'graph']" routerLinkActive="active" style="cursor: pointer;"
```

Comment:
Is this just formatting changes?  If this is unrelated to your goal, can you revert this so the changes in this PR are more scoped?  Maybe clean this up separately (if it's needed, I'm not sure what the coding convention is here).

Reply:
These are all changes that will happen again if someone runs auto-format on the file. No idea if the defaults in code are correct for it though, so I rolled them back.